pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -0,0 +1,1150 @@
+/*
```

Comment:
OK. A general suggestion about this file. Let's move the long methods that populating the configs to a separate file in this package. This will make the lifecycle part move obvious in this file.